### DISCUSSION TITLE
how to override the default solutions from ABSOLUTE

### DISCUSSION THREAD

**Discussion by wunalahu** [ShixiangWang/self-study#38] [Q&A] [answered]
怎样报告自己选的结果呢？absolute的CreateReviewObject函数只选择得分最高的输出到sample.PP-calls_tab.txt文件中，但如果我想选择排序第二的结果输出到sample.PP-calls_tab.txt中，那怎么操作呢？如下图中，得分最高的纯度100%，明显不正确；我想选择排序第二的，但不知道怎么输出到sample.PP-calls_tab.txt文件中。

**Comment by ShixiangWang** [comment, marked as answer]
请仔细阅读官方文档

https://software.broadinstitute.org/cancer/cga/absolute_run

注意⚠️跟你问题相关的最重要部分如下：

![image](https://user-images.githubusercontent.com/25057508/97530772-c9e36a80-19ed-11eb-9420-e3b8e4e7bba2.png)

**Comment by wunalahu** [comment]
谢谢回复，已解决~
















在 2020-10-29 13:51:38，"Shixiang Wang" <notifications@github.com> 写道：

请仔细阅读官方文档

https://software.broadinstitute.org/cancer/cga/absolute_run

注意⚠️跟你问题相关的最重要部分如下：
